WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -150,3 +150,60 @@ $venue-map-wrapper-height: 400px;
 .big-column {
   width: 100%;
 }
+
+.competition-select.form-inline {
+  background-color: #fff;
+  width: 100%;
+  padding-bottom: 10px;
+
+  input,
+  select {
+    font-size: 12px;
+    font-weight: normal;
+    display: block;
+  }
+
+  button {
+    width: 100px;
+  }
+
+  .btn-group {
+    display: block;
+  }
+}
+
+.same-line {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+.competition-list {
+  li:nth-child(odd) {
+    background-color: $table-bg-accent;
+  }
+
+  .date {
+    float: right;
+    font-weight: bold;
+  }
+}
+
+.competition-info {
+  p {
+    margin: 0;
+  }
+
+  // For the on-going competitions, we use the full width of the parent, so it doesn't
+  // make sense to put the <p> on top of each other. Not sure how this will look on mobile.
```

Comment:
I partially agree with Jeremy's suggestion here.

For large screens, I prefer the more vertically compact (all on one line) form. For medium screens, how about a 2-line format? And for small screens, the 3-line format seems best.

There could also be a demarcation between the different p's when combining lines, so perhaps ::after{ content:"-"; } and some spacing. Or something like that.